cli/preset/cloud_preset.go

```diff
@@ -179,7 +200,18 @@ global:
       secretKey: %s
   cloud:
     enabled: true
-    secretName: %s
+    resourceId:
+      secretName: %s
+      secretKey: %s
+    clientId:
+      secretName: %s
+      secretKey: %s
+    clientSecret:
+      secretName: %s
+      secretKey: %s
+    %s
```

Comment:
here are the placeholders for the optional configs.  when they are not set, the call to `getOptionalSecretFromHCPConfig` will return an empty string.